systests/microprofile/client/weld/pom.xml

```diff
@@ -201,6 +171,13 @@
                         </configuration>
                     </execution>
                 </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.github.tomakehurst</groupId>
+                        <artifactId>wiremock-standalone</artifactId>
+                        <version>2.27.2</version>
```

Comment:
@reta  Can this be upgraded to 2.35.0 ? From https://github.com/automatictester/wiremock-maven-plugin/releases, it looks 7.3.0 can work with 2.35.0.

Reply:
@jimma totally makes sens but sadly `microprofile-rest-client-tck` still depends on `javax.servlet` [1], I was thinking of submitting the pull request there first (since we already have `wiremock` beta release).

[1] https://github.com/eclipse/microprofile-rest-client/blob/master/tck/src/main/java/org/eclipse/microprofile/rest/client/tck/sse/MyEventSourceServlet.java#L22